### ISSUE TITLE
Occasional invalid values

### ISSUE THREAD

**Issue by danielsmith89** [unlobito/ha-hildebrandglow#54]
It seems our electricity meter occasionally returns an invalid consumption value of "FFFFFFFFFFFF". Would it be possible to filter out these invalid values so it doesn't affect the consumption values in HA?

Thanks

**Comment by townsmcp** [comment]
Is this error your seeing same as what I have posted in https://github.com/unlobito/ha-hildebrandglow/issues/53 ?

**Comment by danielsmith89** [comment]
> Is this error your seeing same as what I have posted in https://github.com/unlobito/ha-hildebrandglow/issues/53 ?

I don't think so, I'm not losing connection to MQTT or the meter, it's the meter reporting an invalid value for a split second.
